src/api/app/views/layouts/webui/webui.html.erb

```diff
@@ -39,6 +39,7 @@
     <%= csrf_meta_tag %>
   </head>
   <body>
+    <%= render 'peek/bar' %>
```

Comment:
But I would also like to render it in production in the future but therefore we need to somehow label users as "staff".